docs/docsite/rst/guide_aap.rst

```diff
@@ -0,0 +1,165 @@
+.. _ansible_collections.cloud.terraform.docsite.guide_aap:
+
+*************************************************************
+Using cloud.terraform in Ansible Automation Platform
+*************************************************************
+
+The ``cloud.terraform`` collection requires some extra configuration to run in `Ansible Automation Platform (AAP) <https://www.redhat.com/en/technologies/management/ansible>`_; in particular the default execution environment in AAP does not include a Terraform binary due to Terraform licensing restrictions. You must create an execution environment containing a Terraform binary in order to run any Terraform commands from within AAP.
```

Comment:
Wouldn't it make sense to include a link with more information for Terraform's license?

Reply:
I added a link to the Hashicorp licensing FAQ since that seemed more informative than just linking to the Terraform license directly but lmk what you think.